packages/realm/bindgen/vendor/bindgen-lib/CMakeLists.txt

```diff
@@ -148,7 +150,7 @@ set(GENERATED_JS_FILES
     ${TYPESCRIPT_OUTPUT_DIR}/native-rn.mjs
 )
 
-set(NPX ${NODE_FORCE_COLOR} npx)
+set(NPX ${NODE_FORCE_COLOR} npx --prefix .) # without a prefix, npx changes the current directory and that is really confusing and hard to debug
```

Comment:
Meh. Maybe, but it is describing the behavior that we are opting-out of, so it probably isn't that useful to know about, since its hard to imagine someone wanting to go back to it.